tests/dash/test_dash_privatelink.py

```diff
@@ -0,0 +1,101 @@
+import json
+import logging
+from ipaddress import ip_interface
+
+import configs.privatelink_config as pl
+import ptf.testutils as testutils
+import pytest
+from constants import LOCAL_PTF_INTF, LOCAL_DUT_INTF, REMOTE_DUT_INTF, REMOTE_PTF_RECV_INTF, REMOTE_PTF_SEND_INTF
+from gnmi_utils import apply_messages
+from packets import outbound_pl_packets, inbound_pl_packets
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t1'),
+    pytest.mark.skip_check_dut_health
+]
+
+
+"""
+Test prerequisites:
```

Comment:
Since we have the dpuhosts fixture(https://github.com/sonic-net/sonic-mgmt/pull/15695), can we also automate the logic for adding the dpu baisc config? So, we can automate the PL case fully.

Reply:
Yes, I'll be adding this soon!